client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-view/components/motion-manage-timestamp/motion-manage-timestamp.component.ts

```diff
@@ -73,6 +84,18 @@ export class MotionManageTimestampComponent extends BaseUiComponent {
         );
     }
 
+    public ngOnInit(): void {
+        this.subscriptions.push(
+            this.operator.permissionsObservable.subscribe(() => {
+                this._change_metadata = this.operator.hasPerms(
+                    Permission.motionCanManageMetadata,
+                    Permission.motionCanManage
+                );
```

Comment:
```suggestion
                this._change_metadata = this.operator.hasPerms(Permission.motionCanManageMetadata);
```

It is enough to check for motionCanManageMetadata. 
motionCanManageMetadata is the important permission for this field. 
motionCanManage is allowed to be false here

Reply:
done